tests/rptest/transactions/verifiers/stream_verifier.py

```diff
@@ -0,0 +1,322 @@
+
+    def clean_node(self, node):
+        # make sure that there is no such process
+        node.ssh(f"kill -9 $(sudo ps -aux | grep {filename} | "
+                 f"grep -v grep | tr -s ' ' |cut -d' ' -f2)")
+
+    def wait_node(self, node, timeout_sec=sys.maxsize):
+        wait_until(lambda: not (self.is_alive(node)),
+                   timeout_sec=timeout_sec,
+                   backoff_sec=1,
+                   err_msg=f"{title} service {node.account.hostname} failed "
+                   f"to stop within {timeout_sec} sec",
+                   retry_on_exc=False)
+        return True
+
+    # RPCs
+    @staticmethod
+    def _raise_on_non_success(request_result):
+        if request_result.status_code != 200:
+            raise Exception(
+                f"unexpected status code: {request_result.status_code} "
+                f"content: {request_result.content}")
+
+    def _get_ip(self):
+        return self._node.account.hostname
+
+    def _get_url(self, suffix=""):
+        return f"http://{self._get_ip()}:{self._port}/{suffix}"
+
+    def _get(self, url: str, raise_on_fails=False) -> dict:
+        self._redpanda.logger.debug(f"Dispatching GET {url}")
+        try:
+            # Use 120 sec timeout for all requests
+            r = requests.get(url, timeout=120)
+            if raise_on_fails:
+                self._raise_on_non_success(r)
+            return r.json()
+        except (requests.ConnectionError, requests.ConnectTimeout):
+            self.logger.debug(
+                f"{self.service_id} failed to connect to {title} using {url}")
+            return {}
+
+    def _post(self, url: str, payload: dict) -> dict:
+        self._redpanda.logger.debug(f"Dispatching POST {url} with '{payload}'")
+        try:
+            r = requests.post(url, json=payload)
+            self._raise_on_non_success(r)
+            return r.json()
+        except (requests.ConnectionError, requests.ConnectTimeout) as e:
+            self.logger.warning(
+                f"{self.service_id} failed to connect to {title} using {url}: "
+                f"{e}")
+            return {}
+
+    def _delete(self, url: str) -> dict:
+        self._redpanda.logger.debug(f"Dispatching DELETE {url}")
+        try:
+            r = requests.delete(url)
+            self._raise_on_non_success(r)
+            return r.json()
+        except (requests.ConnectionError, requests.ConnectTimeout) as e:
+            self.logger.warning(
+                f"{self.service_id} failed to connect to {title} using {url}: "
+                f"{e}")
+            return {}
+
+    def get_service_config(self):
+        return self._get(self._get_url())
+
+    def update_service_config(self, config):
+        return self._post(self._get_url(), config)
+
+    def get_produce_status(self):
+        return self._get(self._get_url("produce"), raise_on_fails=False)
+
+    def get_consume_status(self):
```

Comment:
nit on naming, make it consistent with "verify" semantics in the other class?

Reply:
yes, will do